app.js

```diff
@@ -1,4 +1,3 @@
-// import functions and grab DOM elements
 import { getRandomThrow, userWins } from './get-random-throw.js';
 
 const playButton = document.querySelector('playButton');
```

Comment:
when using .querySelector() you will need a #. example.
const playButton = document.querySelector('#playButton');